pkg/db/db.go

```diff
@@ -0,0 +1,170 @@
+package db
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sync"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+var (
+	tasks     []*Task
+	tasksMu   sync.Mutex
+	nextID    int = 1
+	taskFile     = "tasks.json"
+)
+
+func Init(dbFile string) error {
+	tasks = []*Task{}
+
+	// Пытаемся загрузить задачи из файла, если он существует
+	if _, err := os.Stat(taskFile); err == nil {
+		data, err := os.ReadFile(taskFile)
+		if err != nil {
+			return fmt.Errorf("ошибка чтения файла задач: %v", err)
+		}
+
+		if len(data) > 0 {
+			err = json.Unmarshal(data, &tasks)
+			if err != nil {
+				return fmt.Errorf("ошибка парсинга задач: %v", err)
+			}
+
+			// Находим максимальный ID
+			maxID := 0
+			for _, task := range tasks {
+				if id := atoi(task.ID); id > maxID {
+					maxID = id
+				}
+			}
+			nextID = maxID + 1
+		}
+	}
+
+	return nil
+}
+
+func saveTasks() error {
+	data, err := json.MarshalIndent(tasks, "", "  ")
+	if err != nil {
+		return fmt.Errorf("ошибка сериализации задач: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("ошибка сериализации задач: %w", err)
```

Reply:
Нужно везде поправить